test/data/css/tests.js

```diff
@@ -1360,9 +1360,8 @@ exports.test_data = {
             '}',
             '.set {',
             '    each(@set, {',
-            '            @{key}-@{index}: @value;',
-            '        }',
-            '    );',
+            '        @{key}-@{index}: @value;',
+            '    });',
```

Comment:
Had to change this test because I believe it is incorrect. It adds extra indents to properties which is unexpected, this test also conflicting with my fixes. I have added example of what it would look like before my changes and after.

Before PR:
```less
@set: {
    one: blue;
    two: green;
    three: red;
}
.set {
    each(@set, {
            @{key}-@{index}: @value;
        }
    );
}
```

After PR:
```less
@set: {
    one: blue;
    two: green;
    three: red;
}
.set {
    each(@set, {
        @{key}-@{index}: @value;
    });
}
```

Reply:
Please refer to the example on the LESS's website which also uses the same formatting as after my changes
https://lesscss.org/functions/#list-functions-each

Reply:
You are correct.  Sometimes the tests document existing wrong behavior so that we'll know when something is fixed.